### PR TITLE
sonar: Use "Arrays.toString(array)" instead

### PR DESCRIPTION
**Pull request by dfa1**
avoiding use of toString() directly on arrays + cleaning loggers

### CODE REVIEW

File: maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/blame/IntegrityBlameCommand.java [hunk 1]
```diff
@@ -123,21 +123,24 @@ private BlameScmResult doShellAnnotate( IntegrityScmProviderRepository iRepo, Sc
         shell.createArg().setValue( "--fields=date,revision,author" );
         shell.createArg().setValue( '"' + filename + '"' );
         IntegrityBlameConsumer shellConsumer = new IntegrityBlameConsumer( getLogger() );
+        String commandLine = CommandLineUtils.toString( shell.getCommandline() );
 
         try
         {
-            getLogger().debug( "Executing: " + CommandLineUtils.toString( shell.getCommandline() ) );
+            if ( getLogger().isDebugEnabled() ) {
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/blame/IntegrityBlameCommand.java]: This isn't necessary anymore as soon as we use SLF4J.

File: maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/diff/IntegrityDiffCommand.java [hunk 2]
```diff
@@ -75,15 +75,17 @@ public DiffScmResult executeDiffCommand( ScmProviderRepository repository, ScmFi
         shell.createArg().setValue( "--filter=changed:all" );
         shell.createArg().setValue( "--filter=format:text" );
         IntegrityDiffConsumer shellConsumer = new IntegrityDiffConsumer( getLogger() );
+        String commandLine = CommandLineUtils.toString( shell.getCommandline() );
 
         try
         {
-            getLogger().debug( "Executing: " + shell.getCommandline() );
+            if ( getLogger().isDebugEnabled() ) {
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/diff/IntegrityDiffCommand.java]: Same here.

File: maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/fileinfo/IntegrityFileInfoCommand.java [hunk 3]
```diff
@@ -74,20 +74,23 @@ public ScmResult executeFileInfoCommand( ScmProviderRepository repository, File
         shell.createArg().setValue( "--user=" + api.getUserName() );
         shell.createArg().setValue( '"' + filename + '"' );
         IntegrityFileInfoConsumer shellConsumer = new IntegrityFileInfoConsumer( getLogger() );
+        String commandLine = CommandLineUtils.toString( shell.getCommandline() );
 
         try
         {
-            getLogger().debug( "Executing: " + shell.getCommandline() );
+            if ( getLogger().isDebugEnabled() ) {
```
Comment (michael-o) [inline maven-scm-providers/maven-scm-provider-integrity/src/main/java/org/apache/maven/scm/provider/integrity/command/fileinfo/IntegrityFileInfoCommand.java]: Same here.

**Comment by dfa1** [comment]
it looks like this PR has been closed without merging. Do I need to update it or reformat code?

**Comment by michael-o** [comment]
It has been merged. Look at the reference commit. I have squashed your commits from two PRs into one.

**Pull request closed without merging**